src/main/features/response/form/models/statement-of-means/employers.ts

```diff
@@ -0,0 +1,71 @@
+import { ValidateNested } from 'class-validator'
+
+import { Serializable } from 'models/serializable'
+import { EmployerRow } from 'features/response/form/models/statement-of-means/employerRow'
+
+export const INIT_ROW_COUNT: number = 1
+export const MAX_NUMBER_OF_JOBS: number = 20
+
+export class Employers implements Serializable<Employers> {
```

Comment:
thoughts on adding a base class which does the majority of this work?
The code is very duplicated

Reply:
or extracting a helper?

Reply:
Yes, I also believe this is a good idea, but I'd rather do it as a separate PR. This one is too big :/

But yes, absolutely we can extract logic to abstract class.